spec/tunes/tunes_stubbing.rb

```diff
@@ -174,7 +174,20 @@ def itc_stub_invalid_update
 def itc_stub_valid_update
   # Called from the specs to simulate valid server responses
   stub_request(:post, "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/apps/version/save/898536088?v=").
-         to_return(status: 200, body: itc_read_fixture_file('update_app_version_success.json'), headers: {'Content-Type' => 'application/json'})
+         to_return(status: 200, body: itc_read_fixture_file("update_app_version_success.json"),
```

Comment:
Line is too long. [95/80]<br>Use 2 (not 7) spaces for indenting an expression spanning multiple lines.